### PR TITLE
Added CODEOWNERS file for the repository

### PR DESCRIPTION
**Pull request by agarcia-oss**
**Description of the change**

This PR includes a CODEOWNERS file for the repository

**Benefits**

Adding this will automatically include the owners as reviewers of any (non draft) PR

**Possible drawbacks**

None.

### CODE REVIEW

**Review by alvneiayu — APPROVED** [review]
LGTM

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+* @juan131 @alvneiayu @mkmik @agarcia-oss
```
Comment (juan131) [inline .github/CODEOWNERS:5]:
```suggestion
* @juan131 @alvneiayu @mkmik @agarcia-oss

```

Comment (juan131) [reply .github/CODEOWNERS:5]: The linter is complaining about a missing newline at the end of file

**Review by juan131 — COMMENTED** [review]
thanks @agarcia-oss ! I'm okay with the changes but please fix the linter warning.

File: .github/CODEOWNERS (hunk 1 above)
Comment (agarcia-oss) [reply .github/CODEOWNERS]: This should be all right now?

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**